packages/Alert/src/alert.scss

```
@@ -1,39 +1,39 @@
-@import '~@igloo-ui/tokens/dist/base10/variables';
-@import '~@igloo-ui/tokens/dist/fonts.css';
-@import './mixins';
+@use '~@igloo-ui/tokens/dist/base10/variables';
```

Je remplacerais le scope de l'import pour spécifier que se sont des tokens. 

```suggestion
@use '~@igloo-ui/tokens/dist/base10/variables' as tokens;
```

---

Voilà, j'ai appliqué ce changement partout